_rules/SC1-2-3-video-alternative.md

```diff
@@ -0,0 +1,182 @@
+---
+name: video with audio has an accessible alternative
```

Comment:
I think "accessible alternative" is too broad because the rule only checks for one aspects of accessibility. Maybe "video with audio has audio descriptions or transcript"?

Reply:
Great suggestion. We have implemented this.